src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    static final int MAX_NUMBER = 100;
```

Comment:
The access modifier for the `MAX_NUMBER` constant is missing. It's a good practice to define it explicitly, usually as `private` if it's only used within the class.